docs/code_coverage.md

```diff
@@ -72,6 +72,17 @@ via `--corpus-dir` to be used instead of the corpus downloaded from OSS-Fuzz:
 python infra/helper.py profile --fuzz-target=<fuzz_target_name> --corpus-dir=<my_local_corpus_dir> $project_name
 ```
 
+### Additional arguments for `llvm-cov`
+
+You might want to use some of the options of [llvm-cov tool] such as
+`-ignore-filename-regex`. You can pass those to the helper script via
+`--extra-args` flag:
+
+```bash
+python infra/helper.py profile --extra-args='-ignore-filename-regex=.*code/to/be/ignored/.*' $project_name
```

Comment:
I think a common convention with command line flags is to use "--" to denote that everything afterwards should be forwarded to a target program. 

I think argparse should support this also: https://stackoverflow.com/questions/14693718/how-to-parse-positional-arguments-with-leading-minus-sign-negative-numbers-usi

Reply:
That's an excellent suggestion, thanks Oliver! Done.